docs/preview features/orchestrator.md

```diff
@@ -0,0 +1,136 @@
+- Cross-platform Support: Runs on Windows, Linux and Mac OS X, including Cloud platforms
+- Free to use: Language models are deployed alongside your bot, there are no usage fees
+
+## Getting Started
+------------------
+### Limitations
+For this preview release, please be aware of these limitations:
+
+1. Orchestrator has to be deployed on an x64 platform. If you are deploying your bot to a cloud service, please be aware that you need to publish your bot with `win-x64` enabled, and enable x64.
+
+1. We are supporting English-only for this release.
+
+1. Orchestrator does not support Entity extraction for this release. Any Entities declared in your LU file will be ignored.
+
+Please see the [roadmap](https://github.com/microsoft/botframework-sdk/blob/main/Orchestrator/docs/Overview.md#Roadmap) for when these features will be added to Orchestrator.
+
+
+### Adding Orchestrator to a new bot in Composer
+1. Enable the preview feature in Orchestrator:
+    - Navigate to `Application Settings` (
+    <img src="orchestrator-assets/application-settings.png" alt="drawing" width="30"/> icon)
+    - Enable Orchestrator preview feature under `Applicatio Updates -> Preview features`
+
+      ![orchestrator flight](orchestrator-assets/preview-flag-orchestrator.png)
+
+1. Create a new bot, or open an existing bot project. Select a dialog in the bot project that you would like to try Orchestrator with. Select `Orchestrator recognizer` from the `Recognizer Type` dropdown
+
+   <img src="orchestrator-assets/orchestrator-recognizer-dropdown.png" alt="recognizer dropdown options" width="300">
+
+1. Create an intent trigger
+    - Click on the waffle menu on a dialog to add a new trigger
+    ![add a trigger](orchestrator-assets/add-trigger.png)
+    - Create a new `Intent recognized` trigger, and fill in the intents for your dialog in LU format. For more details on the LU format, please see [here](https://docs.microsoft.com/en-us/azure/bot-service/file-format/bot-builder-lu-file-format?view=azure-bot-service-4.0)
+    - Add a response to the new `Intent Recognized` trigger to test that Orchestrator is working
+        - In the flow diagram for the newly created trigger, click on the `+` button and select `Send a Response` from the dropdown, then add a sample response in the `Language Generation` textbox.
+
+1. Modify the runtime to support Orchestrator
+    1. Eject the runtime to add Orchestrator libraries to the runtime
+        1. Navigate to `Application Settings` (
+            <img src="orchestrator-assets/application-settings.png" alt="drawing" width="30"/>) and toggle the `Use custom runtime` switch. Then click on `Get a new copy of the runtime code` and select `C#` to eject a copy of the dotnet runtime for your bot. Click `Okay`.
+
+            ![eject-runtime](orchestrator-assets/eject-runtime.png)
+        1. The runtime will be ejected into the `<Bot Project Folder>/runtime/azurewebapp` folder of your  bot project.
```

Comment:
"The runtime will be ejected into the `<Bot Project Folder>/runtime/azurewebapp` folder of your bot project."
An extra space.